x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/MigrateAction.java

```diff
@@ -109,7 +108,7 @@ public boolean isSafeAction() {
 
             BranchingStep conditionalSkipActionStep = new BranchingStep(preMigrateBranchingKey, migrationKey, nextStepKey,
                 (index, clusterState) -> {
-                    if (skipMigrateAction(phase, clusterState.metadata().index(index))) {
+                    if (skipMigrateAction(clusterState.metadata().index(index))) {
                         String policyName =
                             LifecycleSettings.LIFECYCLE_NAME_SETTING.get(clusterState.metadata().index(index).getSettings());
                         logger.debug("[{}] action is configured for index [{}] in policy [{}] which is already mounted as a searchable " +
```

Comment:
Shall we update the logging message to refer to a "partially mounted index"?

Reply:
Done, 0b9f44a577b384a2f5c1c8a07a6a716c5bcc9a6a